data/repositories.toml

```
@@ -410,6 +410,7 @@ repositories = [
   'github.com/mavoweb/mavo',
   'github.com/mcollina/autocannon',
   'github.com/mdx-js/mdx',
+  'github.com/medusajs',
```

Hey @SimonFischerSE 👋 , thanks for contributing. In order to make sure the issues get fetched, the name of the repository needs to be added. I guess the repository name should be `github.com/medusajs/medusa`. Feel free to apply this suggestion:

```suggestion
  'github.com/medusajs/medusa',
```